src/assets/svg/gear-icon.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="14" height="15" viewBox="0 0 14 15" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M12.8249 4.65375L11.9383 4.49156C11.4548 4.36969 11.1729 3.6825 11.4548 3.2775L11.9383 2.54906C12.0195 2.42719 12.0195 2.22563 11.8981 2.145L10.9713 1.21406C10.85 1.09219 10.6895 1.05187 10.5289 1.17375L9.84387 1.65937C9.40054 1.90219 8.71547 1.61906 8.6352 1.13344L8.47374 0.283125C8.4336 0.121875 8.27214 0 8.15174 0H6.82174C6.6612 0 6.53987 0.121875 6.49974 0.283125L6.33827 1.13344C6.21694 1.61906 5.5328 1.90219 5.1296 1.65844L4.4044 1.17375C4.28307 1.05187 4.0824 1.09219 4.00214 1.21406L3.07534 2.145C2.954 2.22563 2.91387 2.42813 3.0352 2.54906L3.51867 3.2775C3.7604 3.6825 3.47854 4.37062 2.99507 4.49156L2.14854 4.65375C1.988 4.69406 1.86667 4.81594 1.86667 4.97719V6.31313C1.86667 6.435 1.988 6.59625 2.14854 6.63656L2.99507 6.79875C3.47854 6.87937 3.7604 7.5675 3.51774 8.01281L3.0352 8.70094C2.91387 8.86219 2.954 9.02438 3.07534 9.14531L4.00214 10.0763C4.0824 10.1981 4.284 10.1981 4.4044 10.1166L5.1296 9.63094C5.5328 9.34781 6.21787 9.63094 6.33827 10.1166L6.49974 11.0072C6.53987 11.1694 6.6612 11.25 6.82174 11.25H8.15174C8.27307 11.25 8.4336 11.1694 8.47374 11.0072L8.6352 10.1166C8.71547 9.63094 9.40054 9.34781 9.84387 9.63094L10.5289 10.1166C10.6895 10.1981 10.8509 10.1981 10.9713 10.0763L11.8981 9.14531C12.0195 9.02344 12.0195 8.86219 11.9383 8.70094L11.4548 8.01281C11.1729 7.5675 11.4548 6.87937 11.9383 6.79875L12.8249 6.63656C12.9864 6.59625 13.0667 6.43406 13.0667 6.31313V4.97719C13.0667 4.81594 12.9864 4.69406 12.8249 4.65375ZM7.46667 3.11625C8.876 3.11625 10.0053 4.24875 10.0053 5.625C10.0053 7.04062 8.876 8.175 7.46667 8.175C6.09654 8.175 4.96907 7.04062 4.96907 5.625C4.96907 4.24875 6.09654 3.11625 7.46667 3.11625Z" fill="#3F51B5"/>
+<svg width="13" height="12" viewBox="0 0 13 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What is the effect of these two lines of changes to the SVG, and why were they necessary? I'm not saying it's wrong, just looking for clarification.

Just curious, did you edit them in text, or with some sort of SVG editor.

Reply:
There were a few pixels of empty space around the icon which made it look slightly off centered in the buttons.
Figma! I'm more familiar with it than Zeplin, it's great for editing and exporting svgs. I also don't know of any other free svg editors